static/js/map.js

```diff
@@ -1174,7 +1179,7 @@ function setupPokestopMarker(item) {
     }
 
     marker.infoWindow = new google.maps.InfoWindow({
-        content: pokestopLabel(item['lure_expiration'], item['latitude'], item['longitude'], item['pokestop_name']),
+        content: pokestopLabel(item['lure_expiration'], item['latitude'], item['longitude'], item['pokestop_name'], item['pokestop_url']),
```

Comment:
This will most likely break others scanners if they dont store this in the same way. Need to have a conditional for alt otherwise you will get undefined errors.